metadata-ingestion/setup.py

```diff
@@ -346,7 +346,7 @@ def get_long_description():
     "nifi": {"requests", "packaging"},
     "powerbi": microsoft_common,
     "powerbi-report-server": powerbi_report_server,
-    "vertica": sql_common | {"sqlalchemy-vertica[vertica-python]==0.0.5"},
```

Comment:
Updated the dialect to support the latest Vertica features such as projections, Mlmodels and OAuth.

Reply:
Where is that dialect's code - the links in the sidebar here https://pypi.org/project/sqlalchemy-vertica-dialect/ don't seem to be working

Reply:
This link is working for me . The link you have sent is correct .